docs/UserGuide.adoc

```diff
@@ -365,12 +370,13 @@ Find all the expenses under the "School" category, with "lunch" tags and dates o
 ====
 * At least one keyword should be provided
 * Keywords are case insensitive, i.e. the KEYWORD `have` is equivalent to the KEYWORD `Have`
-* Keywords can contain multiple words, i.e. the KEYWORD `have lunch` are permitted
+* Keywords can contain multiple words, i.e. the KEYWORD `have lunch` is permitted
 * The order of the keywords does not matter, i.e. the KEYWORD `Have lunch` is equivalent to the KEYWORD `lunch have`
-* Partial words will be matched, i.e. using the KEYWORD `Hav` for expense name will an expense with the name `Have`
+* Partial words will be matched, i.e. using the KEYWORD `Hav` for expense name will give an expense with the name `Have`
+* Only `Tag` keywords can have multiple keywords. `Name`/`Category`/`Cost`/`Date` can only have one keywords.
```

Comment:
Maybe you can give an example of this? i think the user would be quite confused cause a keyword can have multiple words but only tag can have multiple keywords.

then... change the last part to... Name, Category, Cost _and_ Date can only have one _keyword_